tools/analysis_tools/analyze_logs.py

```diff
@@ -124,6 +93,16 @@ def add_plot_parser(subparsers):
         nargs='+',
         default=['mAP_0.25'],
         help='the metric that you want to plot')
+    parser_plt.add_argument(
+        '--start-epoch',
+        type=str,
+        default='1',
+        help='the epoch that you want to start')
+    parser_plt.add_argument(
+        '--eval-interval',
```

Comment:
are you sure the added arguments `start-epoch` and `eval-interval` are used？

Reply:
Removed unused `start-epoch`, and `eval-interval` is used to plot evaluation metric.